claat/proto-renderer/testing-utils/proto_constructors.go

```diff
@@ -34,13 +34,6 @@ func NewStylizedTextStrongAndEmphasized(txt string) *tutorial.StylizedText {
 	}
 }
 
```

Comment:
Colocation nit. `NewLink` moved right before to its `oneof` counterpart, `NewInlineContentLink`